### ISSUE TITLE
Functions hang if you remove the nfc tag in the middle

### ISSUE THREAD

**Issue by Alabate** [Alabate/node-freefare#1]
If you remove the tag in the middle of a transmission, function will never succeed or even fail (unless you put the original tag on the reader) and if you try to check for new tags, it will corrupt spi transmission (with pn532 via spi)

Proposed solutions:

* Bind the nfc_abbort_command to let user cancel when he want
* Automatically abbort command after a timeout, so we can have an userfriendly lib. But timeout have to be editable because transmission can be slower with some protocols.

**Comment by Alabate** [comment]
It seems to happend only with `tag.open()`.

Note: ef025f6 Binds the abort command, but no automatic abort.

**Comment by antroseco** [comment]
I'm having the same issue, even without `tag.open`, I'm just running constantly `device.listTags`.
@Alabate did you find a solution other than calling abort after a timeout?

**Comment by Alabate** [comment]
No, you have to call `abort` because the communication has begun.
